src/components/common/Layout/components/Headline.tsx

```diff
@@ -0,0 +1,54 @@
+import { Box, HStack, Heading, VStack } from "@chakra-ui/react"
+import { useThemeContext } from "components/[guild]/ThemeContext"
+import { ReactNode } from "react"
+
+type Props = {
+  image?: JSX.Element
+  title?: JSX.Element | string
+  description?: JSX.Element
+  action?: ReactNode | undefined
+}
+
+const Headline = ({ image, title, description, action }: Props) => {
+  const { textColor } = useThemeContext()
+
+  return (
+    (image || title || description) && (
```

Comment:
We can remove this condition and make title required, it was for pages like the NFT minting page where we don't want this section at all, now we just won't include the Headline component there